adal/src/androidTest/java/com/microsoft/aad/adal/Util.java

```diff
@@ -205,4 +208,18 @@ static void prepareMockedUrlConnection(final HttpURLConnection mockedConnection)
         Mockito.doNothing().when(mockedConnection).setConnectTimeout(Mockito.anyInt());
         Mockito.doNothing().when(mockedConnection).setDoInput(Mockito.anyBoolean());
     }
+
+    static Map<String, byte[]> getSecretKeys() throws NoSuchAlgorithmException, InvalidKeySpecException, UnsupportedEncodingException {
```

Comment:
This function could use some javadoc -- unclear what this does or who's using it.